src/contexts/TranslationContext.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { LANGS } from "../types/TranslationTypes";
```

Comment:
This is some weird naming. I suggest to use something like 
'locales' or 'languages' just for better readability sake

Reply:
yep, `LANGS` changed to `languages`